<launch>

     <node pkg="sensors_simulator" name="bot1" type="ros_vrpn_client" args="_vrpn_server_ip:=192.168.1.101" required="true" />
     <node pkg="sensors_simulator" name="bot2" type="ros_vrpn_client" args="_vrpn_server_ip:=192.168.1.101" required="true" />

     <node name="opti_data_publisher"
           pkg="sensors_simulator"
           type="opti_pose_data"
           respawn="true"
           output="screen" >
     </node>

     <node name="opti_range_pubber"
           pkg="sensors_simulator"
           type="opti_range_sim"
           respawn="true">
     </node>

</launch>
